sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -218,6 +221,7 @@ export interface JsonWebKey {
     x?: Uint8Array;
     y?: Uint8Array;
 }
+export { JsonWebKey_2 as JsonWebKey }
```

Comment:
this is bizarre to me

Reply:
It's not a bug, but it is the reason #9410 is stalled...  I believe this is the change that introduced this https://github.com/microsoft/rushstack/pull/1767

My understanding is that names that will conflict / shadow declarations in dom.d.ts, etc will be renamed as such

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/9410#issuecomment-641626286

But it's super pervasive - any clash will cause this noise...

Reply:
https://github.com/microsoft/TypeScript/blob/cec2fda9a53620dc545a2c4d7b0156446ab145b4/lib/lib.webworker.d.ts looks like it's getting declared here, which makes me wonder if this is a common browser type, should we even be exporting this? Or simply declaring a shim?

Reply:
https://github.com/microsoft/TypeScript/blob/b2f9432c4064c68862f992b750d7b40c64f194b4/lib/lib.dom.d.ts#L633

Reply:
It's unfortunately not that simple - this JsonWebKey type that we declare here is not conformant with IETF standard for JWK... I believe since the full set is all optional the difference for us is `keyOps` instead of `key_ops`... other languages have more differences but for us I think that's what we would need.

But we already GA'd with this shape...

Reply:
since this is just the interface name, can we maybe rename it so it doesn't conflict anymore, but still export the conflict as deprecated alias?

Reply:
I can, sure. But I do want to call out that if we want to upgrade api-extractor for everyone we will keep bumping into this. Some are already showing this issue (like Error_2, Location_2, etc)  so I'm not sure what the answer would be globally. Any thoughts?

Reply:
Created https://github.com/Azure/azure-sdk-for-js/issues/17710 to address this